inc/saimacsec.h

```diff
@@ -118,31 +118,43 @@ typedef enum _sai_macsec_attr_t
     SAI_MACSEC_ATTR_SCI_IN_INGRESS_MACSEC_ACL,
 
     /**
-     * @brief Indicates if 32-bit Packer Number (PN) is supported.
+     * @brief List of supported cipher-suites
+     *
+     * @type sai_s32_list_t sai_macsec_cipher_suite_t
+     * @flags READ_ONLY
+     */
+    SAI_MACSEC_ATTR_SUPPORTED_CIPHER_SUITE_LIST,
+
+    /**
+     * @brief Indicates if 32-bit Packer Number (PN) is supported.  This is To be deprecated
```

Comment:
there is a flag "@depreacated true" which can be added to flags

Reply:
Done